en-US/firefox-ios.xliff

```diff
@@ -1787,6 +1787,11 @@
         <target>What’s New</target>
         <note>The title for the option to view the What's new page.</note>
       </trans-unit>
+      <trans-unit id="Menu.ZoomPage.Title.v113" xml:space="preserve">
+        <source>Zoom (%@)</source>
```

Comment:
How is the percentage formatted currently?

Internationalization libraries have [that information](https://unicode-org.github.io/cldr-staging/charts/latest/by_type/numbers.number_formatting_patterns.html#17acd127f9139476). If they're using one, and not just doing `X%` (replacing X with a number), then it would be OK.

Reply:
Thank you Flod.
CC @PARAIPAN9 and/or @q2r5 (I think one of you landed that string): could you help confirm the percentage format and if it's currently using an internationalization library? Just want to be extra cautious. thanks

Reply:
The alternative is to actually expose the percentage (`Zoom (%@%)`), but given that symbol is used for the variable placeholder, I expect lots of errors…

Reply:
Hello @flodolo, @Delphine, in code, we are using this format: "%.0f%%"

Reply:
Which is wrong for several locales (including French, German, and Spanish). See previous link for all locales using `#,##0 %`, with a space before the symbol.

Reply:
Hi @PARAIPAN9, gentle ping to see if you were able to update the string, and so we can expose them to localization. Thank you!

Reply:
It seems like a fix was made for this, and I missed it being referenced in this PR...
I synced in with @bcolsson for this, since I'm still trying to understand the fix.
I'll merge later today unless something comes up. Thanks all

Reply:
I went through the swift code (and docs) and I understand very little, so I'll merge this and will monitor once the string lands that things work now as expected.